packages/e2e-tests/helpers/authentication.ts

```diff
@@ -11,14 +11,13 @@ export const E2E_USER_1_API_KEY = "ruk_jukvxbSz7syp4Tw21RzEwSK2bjucNDklCEDmVkjbH
 // trunk-ignore(gitleaks/generic-api-key)
 export const E2E_USER_2_API_KEY = "ruk_K77rBZ4FuPyfj5ocfpPpTXJx7WGVesSySInnCJ4sS1Y";
 
-// Can view the "FE Test Golden Recording" (Test Suites) workspace
-// but cannot upload new recordings nor source-maps
-// trunk-ignore(gitleaks/generic-api-key)
-export const E2E_USER_3_API_KEY = "rwk_FnFcJoJ158eTLXJTg8gpXMto7WT3N8dhNKcAlo2cSHO";
-export const E2E_USER_3_TEAM_ID = "dzoxNzlkZGU4Mi00MmVhLTRkZTctYmI1OC04ZWY0NDQxZTcyYTg=";
-
 // Can view the "Replay Devtools Snapshots" (Test Suites) workspace
 // but cannot upload new recordings nor source-maps
 // trunk-ignore(gitleaks/generic-api-key)
 export const E2E_USER_4_API_KEY = "rwk_CaQCN8h7Wz794K1Yx2200FOcta24j7NkJaYO07TNndV";
 export const E2E_USER_4_TEAM_ID = "dzpmMDZkNWI5OS05MWYzLTRhYTktYTYyNC0zMDRjYjJlYjBlYzk=";
+
+// With only read:workspace role
+// trunk-ignore(gitleaks/generic-api-key)
+export const TEST_RUN_WORKSPACE_API_KEY = "rwk_ONquDEbGGXrjs0yvuON1BhSiAUdlAz8CAl7BlB4igHo";
+export const TEST_RUN_WORKSPACE_TEAM_ID = "dzplODhmZjY1Mi0wYTdmLTRhZDgtOTNiYS04OTZmMTBjYjRjM2Q=";
```

Comment:
Why did these change? Should this change be merged to main?

Reply:
These didn't change. They are new values for the test suite e2e tests